docs/sources/administration/data-source-management/team_ldap.md

```diff
@@ -0,0 +1,130 @@
+
+Configuring multiple rules for a team acts the same way as CAP rules, each rule is evaluated separately. If a team has two rules configured for it, both rules will be applied to the request and the result will be the [logical disjunction](https://en.wikipedia.org/wiki/Logical_disjunction), a "OR" operation of the two rules.
+
+Only a data source administrator with permission access can edit LBAC rules at the data source permissions tab. Changing LBAC rules requires the same access level as editing data source permissions (admin permission for data source).
+
+> "Can I use CAPs (cloud access policies) together with TeamLBAC rules?"
+No, CAP always have precedence. If there are any CAP configured for the same datasource and there are TeamLBAC rules configured, then only the CAP access will be applied.
+
+> "If administrator forget to add rule for a team, what happens?"
+The teams that does not have a rule applied to it, would be able to query the logs if `query` permissions are setup for their role within Grafana.
+
+#### Best practices
+For configuring new rules, or viewing the rules for the teams we recommend testing the rules in the Loki Explore view. This will allow you to see the logs that would be returned for the rule.
+
+
+#### Scenarios
+**Scenario 1: One rule setup for a Team**
+We have two teams, Team A and Team B. Loki access is setup with `Editor`, `Viewer` roles to have `Query` permission.
+
+Team A has a rule `namespace="auth"` configured. 
+
+Team B does not have a rule configured for it. 
+
+**Log Access**
+Team A will have access to logs that match `namespace="auth"`.
+
+Team B will have access to all logs (due to the query permission for the user).
+
+**Scenario 2: Multiple rules setup for two teams**
+We have two teams, Team A and Team B. Loki access is setup with `Admin` roles having `Query` permission.
+
+Team A has rule `namespace="auth", namespace="security"` configured. 
+
+Team B has rule `namespace!="auth", namespace="security"` configured. 
+
+**Log Access**
+Team A will have access to logs that match `namespace="auth"` `OR` `namespace=security`.
+
+Team B will have access to logs that match `namespace!="auth"` `OR` `namespace=security`.
+
+Any admin still have access to query all logs (due to the query permission for the user).
+
+**Scenario 3: Two or more rules configured for a team**
+We have three teams, Team A, Team B and Team C. Team C is a new team and has no rules configured for it. Loki access is setup with `Editor` role having `Query` permission.
+
+Team A has a rule `namespace="auth"` configured.
+
+Team B have two rules `namespace="auth"` and another rule `namespace="security"`. 
+
+Team C have no rules. Team C has a viewer and a editor.
+
+**Log Access**
+Team A will have access to logs that match `namespace="auth"`.
+
+Team B, would be able to query all logs that match either `namespace="auth"` OR `namespace="security"`.
+
+Team C's `Editors`, will have access to query all logs.
```

Comment:
If user is member of both team C and team B, then access is limited to team B rules.